api/api.yaml

```
@@ -1236,6 +1306,27 @@ definitions:
         type: boolean
         example: true
         description: SkyWayのTURNサーバを利用する場合はtrueにします。デフォルトはtrueです
+      credential:
+        $ref: '#/definitions/PeerCredential'
+
+  PeerCredential:
+    type: object
+    required: [ "timestamp", "ttl", "authToken"]
+    properties:
+      timestamp:
+        description: 現在のUNIXタイムスタンプです
+        type: integer
+        format: int64
+        example: 1582789414
+      ttl:
+        description: TTL(600~90000の秒数で指定)。timestamp + TTL の時刻にcredentialが失効します。

```

```suggestion
        description: クレデンシャルの生存期間です。timestampからttl秒経過したとき、クレデンシャルが失効します。
```
credentialはクレデンシャルに統一しましょう。
引数の値域は `minimum` や `maximum` を使うのはいかがでしょうか。

---

> 引数の値域は minimum や maximum を使うのはいかがでしょうか。

引数の値域とは、何のことを指しているのでしょうか?

---

整数型（int64）の引数 `ttl` の、有効な値の範囲 `[600, 9000]` を指して値域という言葉を使いました 。オレオレ用語ですいません 🙇